components/LetterTile.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react'
+
+export default class LetterTile extends Component {
+  render() {
+    return (
+      <div>
```

Comment:
**Readiness for change:** I might consider adding a `className` property here or something so that it is easier to apply css styling